components/clientComponents/forms/Review/FormElements/BaseElementArray.tsx

```diff
@@ -0,0 +1,36 @@
+import { FormItem } from "../helpers";
+
+export const BaseElementArray = ({
+  formItem,
+  splitValues,
+}: {
+  formItem: FormItem | undefined;
+  splitValues?: boolean;
+}): React.ReactElement => {
+  if (!formItem) {
+    return <></>;
+  }
+
+  return (
+    <div className="mb-8">
+      <dt className="mb-2 font-bold">{formItem.label}</dt>
+      <dd>
+        {Array.isArray(formItem.values) ? (
```

Comment:
Should we catch this higher up ?

  if (!formItem || !Array.isArray(formItem.values)) {